azure-devops/global/stages/50-deployment/dynamic.yaml

```diff
@@ -1,19 +1,17 @@
-parameters:
-  - name: 'RESOURCE_GROUP_NAME'
-    type: 'string'
-    default: ''
-
 steps:
+  - script: |
+      RESOURCE_GROUP_NAME="${ENVIRONMENT}-rg-${RESOURCE_GROUP_NAME}-${AZM_LOCATION}"
```

Comment:
You are creating here, ok. It's kinda inconsistent. Please stabilize the consistency. Are the variables going to be uppercase or lowercase?